src/comlink.ts

```diff
@@ -371,6 +375,50 @@ function throwIfProxyReleased(isReleased: boolean) {
   }
 }
 
+function releaseEndpoint(ep: Endpoint) {
+  return requestResponseMessage(ep, {
+    type: MessageType.RELEASE,
+  }).then(() => {
+    closeEndPoint(ep);
+  });
+}
+
+interface FinalizationRegistry<T> {
+  new (cb: (heldValue: T) => void): FinalizationRegistry<T>;
+  register(
+    weakItem: object,
+    heldValue: T,
+    unregisterToken?: object | undefined
+  ): void;
+  unregister(unregisterToken: object): void;
+}
+declare var FinalizationRegistry: FinalizationRegistry<Endpoint>;
+
+const proxyCounter = new WeakMap<Endpoint, number>();
+const proxyFinalizers =
+  "FinalizationRegistry" in self &&
```

Comment:
This PR cannot work in the nodejs worker environment, because `self` is used here, and there is no such object in the nodejs environment, which will cause the following error:

```
ReferenceError: self is not defined
```

Reply:
v4.4.0 can no longer work in the nodejs environment.

Reply:
Oh dear, I will get right on this. We really need a nodejs test harness.

Reply:
4.4.1 has now beeen published with a fix and a unit test to prevent this regression in future.